pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -239,6 +242,7 @@ def __init__(self, *args, **kwargs):
         self.test_id = kwargs.get('test_id', self.root)
         self.eval_attr = kwargs.get('eval_attr', None)
         self.xdist_ip_addresses = kwargs.get('xdist_ip_addresses', None)
+        self.processes = kwargs.get('processes', None)
```

Comment:
This should probably be updated to use the same `0` default as above and work for local use of multiple workers.